curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-styles-based-on-data.md

```diff
@@ -30,55 +28,73 @@ Add the `style()` method to the code in the editor to set the `color` of the `h2
 The first `h2` should have a `color` of red.
 
 ```js
-assert($('h2').eq(0).css('color') == 'rgb(255, 0, 0)');
+const headingTwoFirst = document.querySelector('h2');
+const color = window.getComputedStyle(headingTwoFirst)['color'];
+assert.strictEqual(color, 'rgb(255, 0, 0)');
 ```
 
 The second `h2` should have a `color` of green.
 
 ```js
-assert($('h2').eq(1).css('color') == 'rgb(0, 128, 0)');
+const headingTwoSecond = document.querySelectorAll('h2')[1];
+const color = window.getComputedStyle(headingTwoSecond)['color'];
```

Comment:
```suggestion
assert.isNotNull(headingTwoSecond);
const color = window.getComputedStyle(headingTwoSecond)['color'];
```

can I leave you to adjust in the other places?

Reply:
Hey Ilenia, I hate to be rude but can you re-review it? I had a git commit I forgot to push that mostly resolves it. While I did use `exists` instead of `isNotNull` , it should be functionally identical.

Reply:
ok, that will be tomorrow tho